src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -368,14 +374,24 @@ static inline void cavs_pm_runtime_dis_dsp_pg(uint32_t index)
 		if (tries == 0)
 			tr_err(&power_tr, "cavs_pm_runtime_dis_dsp_pg(): failed to power up core %d",
 			       index);
+		if (flag)
```

Comment:
Just assign value compited in L346 (TPLG must be used by a client of the API too).